tests/e2e/tests/regression/workbench/editor-cleanup.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { acceptLicenseTerms, acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { WorkbenchPage, MyRedisDatabasePage, SettingsPage } from '../../../pageObjects';
+import { rte } from '../../../helpers/constants';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+import { addNewStandaloneDatabasesApi, deleteStandaloneDatabaseApi, deleteStandaloneDatabasesApi } from '../../../helpers/api/api-database';
+import { Common } from '../../../helpers/common';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const workbenchPage = new WorkbenchPage();
+const common = new Common();
+const settingsPage = new SettingsPage();
+
+const commandToSend = 'info server';
+const databasesForAdding = [
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testDB1' },
+    { host: ossStandaloneConfig.host, port: ossStandaloneConfig.port, databaseName: 'testDB2' }
+];
+
+fixture `Workbench Editor Cleanup`
+    .meta({ type: 'critical_path', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        // Clear and delete database
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    });
+test('Disabled Editor Cleanup toggle behavior', async t => {
+    // Go to Workbench page
+    await t.click(myRedisDatabasePage.workbenchButton);
+    // Send commands
+    await workbenchPage.sendCommandInWorkbench(commandToSend);
+    await workbenchPage.sendCommandInWorkbench(commandToSend);
+    // Verify that Editor input is not affected after user running command
+    await t.expect((await workbenchPage.queryInputScriptArea.textContent).replace(/\s/g, ' ')).eql(commandToSend, 'Input in Editor is saved');
+});
+test('Enabled Editor Cleanup toggle behavior', async t => {
```

Comment:
`Enable` instead of `Enabled`

Reply:
no, it's right
test checks behavior of "Workbench Editor" when "Editor Cleanup" toggle is in enabled state
it's not just about disabling/enabling toggle